cmd/authd-oidc/daemon/daemon.go

```diff
@@ -125,6 +125,12 @@ func (a *App) serve(config daemonConfig) error {
 		}
 	}
 
+	brokerConfigDir := config.Paths.BrokerConf + ".d"
```

Comment:
Mybe we can avoid using constants here by refactoring something shared with https://github.com/ubuntu/authd-oidc-brokers/blob/4227a64cd96c11d954ea3f50828835e739263a5f/internal/broker/config.go#L35 ?